src/WebApp/Startup.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Linq;
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+
+using Codidact.Authentication.Application;
+using Codidact.Authentication.Infrastructure;
+using Codidact.Authentication.Infrastructure.Persistance;
+using Codidact.Authentication.Domain.Entities;
+
+namespace Codidact.Authentication.WebApp
+{
+    public class Startup
+    {
+        private readonly IConfiguration _configuration;
+
+        public Startup(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddRouting(options =>
+            {
+                options.LowercaseUrls = true;
+            });
+
+            services.AddRazorPages()
+                .AddRazorRuntimeCompilation();
+
+            services.AddInfrastructure(_configuration);
+            services.AddApplication(_configuration);
+        }
+
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+            else if (env.EnvironmentName != "Testing")
+            {
+                throw new NotImplementedException(env.EnvironmentName);
+            }
+
+            app.UseStaticFiles();
+            app.UseRouting();
+
+            app.UseIdentityServer();
+            app.UseAuthorization();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapRazorPages();
+            });
+
+            SeedDatabase(app);
+        }
+
+        private static void SeedDatabase(IApplicationBuilder app)
+        {
```

Comment:
Sorry for asking about this 20 times but what is the reason for going back to this instead of opening a web page when there are no users?

Reply:
There has to be at least one user so that it is possible to test if the login works. That is why I added this code originally.

Then, some time ago, I setup a forum software for something else, and thought, wouldn't it be nice to have an interactive installer that sets the page title, creates an administrator, therefor, I created the `/account/firstuse` endpoint.

However, it really makes no sense to do this in the MVP. Setting up such a installer is not trivial and maintaining it would be a very annoying because you would really only use it once anyway.

Therefor, I created issue #3 and rolled back the changes.